src/Item.test.jsx

```diff
@@ -1,28 +1,47 @@
-import { render, fireEvent } from '@testing-library/react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
 
 import Item from './Item';
 
-test('Item', () => {
-  const task = {
-    id: 1,
-    title: '뭐라도 하기',
-  };
+describe('Item', () => {
+  afterEach(() => {
+    cleanup();
+  });
 
-  const handleClick = jest.fn();
+  it('shows task item correctly', () => {
+    const task = {
+      id: 1,
+      title: '뭐라도 하기',
+    };
 
-  const { container, getByText } = render((
-    <Item
-      task={task}
-      onClickDelete={handleClick}
-    />
-  ));
+    const { container } = render((
+      <Item
+        task={task}
+      />
```

Comment:
```suggestion
      <Item task={task} />
```

props가 하나 인 경우에는 한줄로 해도 괜찮을 것 같아요!
줄바꿈해주는 것도 기준을 마련하시면 좋아요 ;)